rest-service/manager_rest/storage/management_models.py

```diff
@@ -588,7 +588,27 @@ def write_ca_cert(self):
         return f.name
 
 
-class Manager(_WithCACert, SQLModelBase):
+class CloudifyNodeMixin(object):
+    """A mixin for Cloudify cluster node objects."""
+
+    @property
+    def name(self):
+        return self.name
```

Comment:
those properties by themselves seem useless because if you do access `.name` then of course it just returns `self.name` so that's just unlimited recursion;  it's only not unlimited where subclasses override this, so i suppose it would make more sense for these methods to just return none

Reply:
Good point, I'll fix it